src/npc/ai/misc.rs

```diff
@@ -2609,4 +2624,9 @@ impl NPC {
 
         Ok(())
     }
+
+    fn chest_has_missile_flag(&self) -> bool {
+        let missile_flags: [u16; 9] = [200, 201, 202, 218, 550, 766, 880, 920, 1551];
+        missile_flags.contains(&self.flag_num)
+    }
```

Comment:
This array should be in engine constants, with the rest of gameplay editable values. Would make it simpler to test `contains` within the chest npc instead of defining a one-line helper function